test/integration/targets/web3-1.0.0/web3.ts

```diff
@@ -10,7 +10,9 @@ export let web3: Web3;
 export let accounts: string[];
 
 export async function createNewBlockchain() {
-  const web3 = new Web3(ganache.provider());
+  const web3 = new Web3(ganache.provider(), undefined, {
+    transactionConfirmationBlocks: 1,
```

Comment:
I didn't look too deeply into this, but without this option, Web3 seemed to hang around for more confirmations.

Reply:
Hmm i saw similar behaviour with *SOME* web3 versions in the past.